pytential/linalg/utils.py

```diff
@@ -353,6 +353,30 @@ def cluster_skeletonization_error(
         mat: np.ndarray, skeleton: "SkeletonizationResult", *,
         ord: Optional[float] = None,
         relative: bool = False) -> np.ndarray:
+    r"""Evaluate the cluster-wise skeletonization errors for the given *skeleton*.
+
+    Errors are computed for all interactions between cluster :math:`i` and
+    cluster :math:`j` as
+
+    .. math::
+
+        E^T_{ij} = \|A_{ij} - L_i T_{ij}\|
+        \quad \text{and} \quad
+        E^S_{ij} = \|A_{ij} - S_{ij} R_j\|
+
+    where :math:`A_{ij}` is the exact interaction between the two clusters,
+    :math:`(L_i, R_j)` are the ID interpolation matrices and
+    :math:`(T_{ij}, S_{ij})` are the target and source skeleton matrices.
+    The exact matrix and the skeleton matrices are extracted from the full
+    matrix *mat*.
+
+    :arg ord: the type of the matrix norm used to compute errors, as described
+        in :func:`numpy.linalg.norm`.
```

Comment:
State that this identifies the norm $\|\cdot\|$ used above.

Reply:
Added a sentence in https://github.com/inducer/pytential/compare/440161e74f559e51120245ce59f8c1a6c4f15d77..0617bdafd0a7a37c2e162b752371ab908e456ad5.